source/en/components/pageview.md

```diff
@@ -0,0 +1,114 @@
+# PageView component reference
+
+The Page view, Use when you need to switch the entire page.
```

Comment:
PageView component is derived from ScrollView, the difference is that when scrolls it automatically snaps to next page of content.